main.yml

```diff
@@ -290,6 +290,31 @@ landing:
   channel: '#flip-mode-squad'
   deployment_repo: https://github.com/RedHatInsights/landing-page-frontend-build
 
+migrations:
+  title: Migrations
+  frontend:
+    sub_apps:
+      - id: migration-analytics
+        default: true
+  top_level: true
+
+migration-analytics:
+  title: Migration Analytics
+  api:
+    versions:
+      - v1
+    alias:
```

Comment:
This means that in the api-docs page we'll show the app as xavier instead of `Migration Analytics` is that correct?

Reply:
We'd like to show `Migration Analytics` instead of `xavier`. I'll add a new commit to change it. Thanks

Reply:
Thanks for the feedback. This PR has the same changes we merged into the qa-stable branch. We will apply the changes that you have suggested starting from ci-stable to validate and promote them.

Reply:
Yeah, sorry that I didn't noticed this before, but it totally got out of my radar.

Reply:
Sorry for the confusion. Myself and Carlos were updating in parallel. Just to reiterate we won't be making a change to this PR. We must release to prod-stable the same configuration that we have tested in ci-stable and qa-stable.

Reply:
Ah, okay. Sure thing! If you want to discuss the docs api feel free to ping me and we can discuss it more deeply.